wdl-analysis/src/handlers/find_all_references.rs

```diff
@@ -0,0 +1,217 @@
+//! Handlers for "find all references" requests.
```

Comment:
This would be a good spot to include a high-level description of what this is. Perhaps include a link to the spec, although it is pretty light on details too: https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_references